### ISSUE TITLE
[Accessibility]Focus is not locked in the dialog when the dialog is firstly opened

### ISSUE THREAD

**Issue by liuguangyicao** [MicrosoftDocs/msteams-docs#7858]
# Description
When the dialog is firstly opened and not clicked, we will find that focus is not locked in the dialog. i.e. the tab focus is going out of the dialog at the background. This is also observed with screen reader quick navigation keys. i.e. the background elements are visible

https://user-images.githubusercontent.com/25132507/211231569-81029eba-ac21-4c8a-8957-820485ec4449.mov

<img width="1431" alt="image2" src="https://user-images.githubusercontent.com/25132507/211232111-80e809ce-6e32-439d-b3fb-3d89083e4304.png">

# Expected Behaviour
Tab focus should be locked in the dialog. When dialogs are open, inspect through Screen reader quick navigation keys and ensure the background elements are not visible

# Actual Behaviour
Tab focus is not locked in the dialog and the background elements are visible.

# Card Json
```
{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.4",
  "body": [
    {
      "id": "topic1",
      "type": "Input.Text",
      "placeholder": "Placeholder",
      "label": "Label",
      "maxLength": 256,
      "isRequired": true,
      "errorMessage": "Test Error Message",
      "spacing": "Large"
    },
    {
      "type": "Input.ChoiceSet",

      "choices": [
        {
          "title": "Choice 1",
          "value": "Choice 1"
        },
        {
          "title": "Choice 2",
          "value": "Choice 2"
        }
      ],
      "id": "userIds",
      "isMultiSelect": true,
      "placeholder": "Placeholder",
      "label": "Label",
      "isRequired": true,
      "errorMessage": "Test Error Message"
    },
    {
      "id": "topic",
      "type": "Input.Text",
      "placeholder": "Placeholder",
      "label": "Label",
      "maxLength": 256,
      "isRequired": true,
      "errorMessage": "Test Error Message",
      "spacing": "Large"
    },
    {
      "type": "ActionSet",
      "actions": [
        {
          "type": "Action.Submit",
          "title": "Send",
          "id": "sendAction",
          "associatedInputs": "auto",
          "data": {
          "path": "/test"
          }
        }
      ]
    }
  ]
}
```

**Comment by ChetanSharma-msft** [comment]
Hello @liuguangyicao - Thanks for reporting your issue.
We will look into it and let you know the updates.

**Comment by Prasad-MSFT** [comment]
@liuguangyicao - Could you please confirm whether you are having this issue in Teams desktop client or web client and in which version of teams?  Could you please confirm which screen reader you are using whether its Narrator, JAWS, or NVDA?

**Comment by Prasad-MSFT** [comment]
We tried to repro this issue using Narrator, however on opening the dialog for the first time the focus is coming to dialog only.
We tested this both in MS Teams desktop client (Microsoft Teams Version 1.6.00.364) and web client. Attaching the video for reference:
[AccessibilityDialog.pptx](https://github.com/MicrosoftDocs/msteams-docs/files/10379950/AccessibilityDialog.pptx)
